packages/data-broker/src/__tests__/services/kobo/KoBoService.stubs.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { MOCK_DB_DATA, MOCK_KOBO_RESULT } from './KoBoService.fixtures';
+
+const mockFind = (array, criteria) => {
+  return array.filter(currentObject => {
```

Comment:
Don't need the redundant return, but up to you if you prefer it staying there

```suggestion
const mockFind = (array, criteria) => array.filter(currentObject => {
```